src/html_css/scripts/app.js

```diff
@@ -0,0 +1,88 @@
+import { debounce } from './tools/debounce.js'
+
+import { photosData } from '../data/photos-data.js'
+
+import { paginator } from './pagination.js'
+import { Slider } from './slider.js'
+import { Storage } from './storage.js'
+
+export class App {
+	init() {
+		// storage init
+		const storage = new Storage(photosData)
+		const storageData = storage.getSliderData()
+
+		// pagination and slider init
+		paginator('.blog__posts', storageData)
+		const slider = new Slider('.prefers__slider', storageData)
+
+		//slick slider init
+		$(document).ready(function () {
+			$('.courses__cards-container').slick({
+				mobileFirst: true,
+				variableWidth: true,
+				slidesToShow: 1,
+				slidesToScroll: 1,
+
+				prevArrow: $('.courses__slide_left'),
+				nextArrow: $('.courses__slide_right'),
+
+				responsive: [
+					{
+						breakpoint: 767,
+						settings: {
+							slidesToShow: 2,
+							slidesToScroll: 1,
+						},
+					},
+					{
+						breakpoint: 1440,
+						settings: {
+							slidesToShow: 3,
+							slidesToScroll: 1,
+						},
+					},
+				],
+			})
+		})
+
+		// save form value in storage
+		const form = document.getElementById('form')
+		const nameInput = form.querySelector('.blog__form-name')
+		const phoneInput = form.querySelector('.blog__form-phone')
+		const emailInput = form.querySelector('.blog__form-email')
+
+		nameInput.addEventListener(
```

Comment:
To separate it all out into a separate function to avoid violating the DRY principle